selftests/unit/test_test.py

```diff
@@ -112,7 +112,6 @@ def test_no_data_dir(self):
         above_limit_name = os.path.join(self.tmpdir, "a" * 251)
         tst = self._get_fake_filename_test(above_limit_name)
         self.assertFalse(tst.get_data('', 'file', False))
-        tst._record_reference       # Should do nothing
```

Comment:
This may have made sense earlier, I think this was an attribute at a given point in time.  But yes, makes sense removing it.